CMakeLists.txt

```diff
@@ -540,6 +540,13 @@ if(ANDROID)
   include_directories(Source/Android)
 endif()
 
+if(NOT CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
```

Comment:
Hmm, this should probably only be for our own code, similar to how C++14/C++17 is only enabled for Source/. I'll change this.